# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/tests/focus/activeelement-after-immediately-focusing-same-site-iframe-contentwindow.html
# Commit: c63ecdad6a3c
# Full Hash: c63ecdad6a3c60ba625517d34bb29c8aae9f5e95
# Author: Henri Sivonen <hsivonen@hsivonen.fi>
# Date: 2021-02-22 17:34:52
# Regressor Bug: 1663931
# File Overlap Count: 1
# Description:
#   Bug 1663931 - Avoid moving focus when changing iframe remoteness. r=nika,mccr8
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D99215
# ==============================================================================

diff -r 828c62bfc0a2 -r c63ecdad6a3c testing/web-platform/tests/focus/activeelement-after-immediately-focusing-same-site-iframe-contentwindow.html
--- a/testing/web-platform/tests/focus/activeelement-after-immediately-focusing-same-site-iframe-contentwindow.html	Mon Feb 22 10:48:53 2021 +0000
+++ b/testing/web-platform/tests/focus/activeelement-after-immediately-focusing-same-site-iframe-contentwindow.html	Mon Feb 22 10:51:51 2021 +0000
@@ -6,19 +6,8 @@
 <script>
 setup({explicit_done:true});
 window.onmessage = function(e) {
-    var actual = e.data;
     test(function() {
-        // Make the difference between Firefox and Chrome visible separately
-        // from the comparison of the entire log string failing to match.
-        var endedWith = false;
-        if (actual.endsWith(",willspineventloop,")) {
-            endedWith = true;
-            actual += "innerbodyfocus,";
-        }
-        assert_true(endedWith, "Should not have gotten innerbodyfocus after willspineventloop");
-    }, "Check trailing events");
-    test(function() {
-        assert_equals(actual, "outerparse,activeElement:BODY,willfocusiframe,didfocusiframe,activeElement:IFRAME,willbluriframe,didbluriframe,activeElement:IFRAME,willspineventloop,innerbodyfocus,", 'Check log');
+        assert_equals(e.data, "outerparse,activeElement:BODY,willfocusiframe,didfocusiframe,activeElement:IFRAME,willbluriframe,didbluriframe,activeElement:IFRAME,willspineventloop,", 'Check log');
     }, "Check result");
     w.close();
     done();